### DISCUSSION TITLE
your pfp is sus

### DISCUSSION THREAD

**Discussion by ItsEnderium** [CluckCluckChicken/Magnifier#28] [Q&A]
your pfp looks like red with one leg

**Comment by davwat** [comment]
![sus drip](https://user-images.githubusercontent.com/46281840/120070674-9fd95000-c083-11eb-9812-d9b463023e23.png)

**Reply by ItsEnderium** [reply]
thats not a rocket, thats the impostor knife

**Reply by davwat** [reply]
dun dun dun dun dun dun dun dududun
